pages/api/search/issues/index.ts

```diff
@@ -150,6 +176,64 @@ async function get(req: NextApiRequest, res: NextApiResponse) {
                                     .map((value)=> value === '+' ? Sequelize.literal('+') : 
                                       (COLS_TO_CAST.includes(value) ? castToDecimal(value) : value));
 
+    if(lastEdited) {
```

Comment:
code complexity on this endpoint has grown beyond control; Would be simpler to add these conditions to their own route - this way we _don't add more_ technical debt.

It would also help identify repeated block-codes that can be refactored into a helper function to further reduce technical debt on the added code